src/modules/Dashboard/features/Applet/Activities/ActivitiesToolbar/ActivitiesToolbar.tsx

```diff
@@ -74,15 +80,17 @@ export const ActivitiesToolbar = ({
               </Button>
             )}
 
-            <Button
-              component={Link}
-              to={generatePath(page.builderAppletActivities, { appletId })}
-              variant="contained"
-              data-testid={`${dataTestId}-add-activity`}
-              sx={{ minWidth: theme.spacing(13.2) }}
-            >
-              {t('addActivity')}
-            </Button>
+            {canEditApplet && (
+              <Button
+                component={Link}
+                to={generatePath(page.builderAppletActivities, { appletId })}
+                variant="contained"
+                data-testid={`${dataTestId}-add-activity`}
+                sx={{ minWidth: theme.spacing(13.2) }}
+              >
+                {t('addActivity')}
+              </Button>
```

Comment:
In addition to this, according to scenario 3 of [M2-6727](https://mindlogger.atlassian.net/browse/M2-6727), it sounds like the review should not be able to see the activities tab either. I'm not sure if that's what was meant, since the language was vague but I thought I'd point that out. Are you clear on this?

In case it turns out they shouldn't, I'm thinking of what the solution would be. In our multi-informant environment we use a pop-up like this to block access to the team tab

<img width="1327" alt="image" src="https://github.com/ChildMindInstitute/mindlogger-admin/assets/14842108/1b4d8fa6-3b57-4185-a4e4-1e5ac1b7a1c4">

However, for the schedule tab I see something like this, which feels a little cleaner

<img width="1323" alt="image" src="https://github.com/ChildMindInstitute/mindlogger-admin/assets/14842108/a03a9729-eef6-41ac-b9f5-ed6c24f5efe2">

Reply:
I wanted to piggy-back off of this comment, because I ran into a couple similar issues. 

I'm not sure if it's within the scope of these tickets, which seem very specific about particular buttons/menu items, rather than access to tabs per-se, but I think it's worth discussing.

While the Applet-level Schedule tab has a nice "Forbidden" empty state for Reviewers, the Schedule tab at the Participant level raises the same "No access to applet" alert and redirects to the applets list. 

It seems like maybe we should be consistent in our behaviour here? I think the "Forbidden" empty states are preferable and less disruptive, but if that's the case then we need updated empty states for the Participants and Teams tab as well — the default empty states for both include buttons prompting the user to "Add a participant/manager", which ought to be conditional based on role.

Reply:
@sultanofcardio went ahead and added permission checks to return the `NoPermissions` component instead of the modal. This should fix the problem you highlighted for Reviewer and Coordinator roles, and also applied the same changes for all Editor tabs

@JustinNusca This wasn't a huge lift so I've added the same behavior in the Participant > Schedule for Reviewers and Editors